packages/contract/lib/reason.js

```diff
@@ -23,14 +23,28 @@ const reason = {
       res && typeof res === "object" && typeof res.result === "string";
 
     if (isObject) {
+      // TODO: ganache 2.0 returns the reason string already when
+      // vmErrorsOnRPCResponse === true. Why not use it?
```

Comment:
```suggestion
      // NOTE that Ganache >=2 returns the reason string when
      // vmErrorsOnRPCResponse === true, which this code could
      // be updated to respect (instead of computing here)
```